.ruby-version

```diff
@@ -1 +1 @@
-2.2.1
+2.3.1
```

Comment:
why not 2.4?

Reply:
answering myself: 2.3 used in other apps, we want parity.

Reply:
I think parity is good - 2.3 is what we use most places, and 2.2 is still supported in rails 5, so no rush to 2.4